backend/src/v5/services/eventsListener/components/modelEvents.js

```diff
@@ -66,12 +67,20 @@ const revisionUpdated = async ({ teamspace, project, model, data, sender }) => {
 
 const revisionAdded = async ({ teamspace, project, model, revision, isFederation }) => {
 	try {
-		const { tag, author, timestamp } = await getRevisionByIdOrTag(teamspace, model, stringToUUID(revision),
-			{ _id: 0, tag: 1, author: 1, timestamp: 1 });
+		const { tag, author, timestamp, description, rFile } = await getRevisionByIdOrTag(teamspace,
+			model, stringToUUID(revision),
+			{ _id: 0, tag: 1, author: 1, timestamp: 1, description: 1, rFile: 1 });
+
 		const event = isFederation ? chatEvents.FEDERATION_NEW_REVISION : chatEvents.CONTAINER_NEW_REVISION;
 
-		await createModelMessage(event, { _id: revision, tag, author, timestamp: timestamp.getTime() },
-			teamspace, project, model);
+		const format = getRevisionFormat(rFile);
+
+		await createModelMessage(event, { _id: revision,
+			tag,
+			author,
+			timestamp: timestamp.getTime(),
+			description,
+			...(format ? { format } : undefined) }, teamspace, project, model);
```

Comment:
use `deleteIfUndefined` from object helper instead
```suggestion
			...(format ? { format } : undefined) }, teamspace, project, model);
```